spec/models/manageiq/providers/embedded_ansible/provider_spec.rb

```diff
@@ -68,4 +68,27 @@
       expect(Hardware.count).to              eq(0)
     end
   end
+
+  context "ensure_managers callback" do
+    before do
+      EvmSpecHelper.local_miq_server(:is_master => true, :zone => Zone.seed)
```

Comment:
I don't think `:zone => Zone.seed` is what you're looking for here.  I think you want to add
```
MiqRegion.seed
Zone.seed
```

[here](https://github.com/ManageIQ/manageiq/blob/5c9be160f538f1ac60547b38eabb59a8355e6189/spec/models/manageiq/providers/embedded_ansible/provider_spec.rb#L7) before the `assign_embedded_ansible_role(miq_server)`